src/main/java/com/microsoft/azure/functions/worker/broker/CoreTypeResolver.java

```diff
@@ -119,4 +120,21 @@ static String getBindingNameAnnotation(Parameter param) {
 		}
 		return new String("");
 	}
+
+    static boolean checkHasImplicitOutput(Parameter parameter){
+        try {
+            Annotation[] annotations = parameter.getAnnotations();
+            for (Annotation annotation : annotations) {
+				if (!annotation.annotationType().getName().contains(Constants.JAVA_LIBRARY_PACKAGE_PREFIX)) continue;
```

Comment:
this will return class qualified name which ends with Class name, I think only check Prefix that match with azure functions is enough. If we insist to check it package name end with annotation it requires string processing, it may affect the performance as well and involving potential bugs.